### PR TITLE
Add missing reference docs for push and bundle

### PR DESCRIPTION
**Pull request by dnephin**

### CODE REVIEW

**Comment by SvenDowideit** [comment]
@sfsmithcha @londoncalling can you guys take a look

at this point, I think the compose docs we've published are still for 1.7 - but this is relevant for (at least 1.12 experimental)

if we were to publish this, we'd also need to add an advisory for compose.

File: docs/reference/bundle.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+```
+
+Usage: bundle [options]
+
+Options:
+    --fetch-digests            Automatically fetch image digests if missing
+
+    -o, --output PATH          Path to write the bundle file to.
+                               Defaults to "<project name>.dsb".
```
Comment (albers) [inline docs/reference/bundle.md]: this was changed to `dab`.

**Comment by sfsmithcha** [comment]
@icecrime PTAL. I understood that because this was experimental that the docs would not be published yet.

**Comment by dnephin** [comment]
I don't think the Compose side is experimental. The output of the commands might change as the bundle format changes, but the command and Compose UX shouldn't need to change in any backwards incompatible way.

**Comment by sfsmithcha** [comment]
Sorry for the confusion @dnephin . Sven is talking about publishing this into the current (1.7) doc set with an advisory because it relates to Engine 1.12 rc that is in the docs.

I don't think there's any question that this should go out  at 1.8 GA as released product for Compose.

**Comment by aanand** [comment]
@dnephin Could you update this now that the `--fetch-digests` flag has changed?

**Comment by dnephin** [comment]
updated

**Comment by aanand** [comment]
LGTM

**Comment by aanand** [comment]
retest this please

**Comment by albers** [comment]
>Could you update this now that the --fetch-digests flag has changed?

@aanand can you please point me to what happened to `--fetch-digests`? Was it just removed?

**Comment by albers** [comment]
@aanand found it: #3762

**Pull request merged**